docs/utilities.md

```diff
@@ -124,4 +124,105 @@ def setup():
 
 Results in:
 
-![dist demo](img/dist_2.png)
\ No newline at end of file
+![dist demo](img/dist_2.png)
+
+### Scale (one parameter)
+
+Scales the canvas units by x horizontally and vertically. 
+
+Usage:
+
+```python
+scale(x)
+```
+**Parameters**
+
+- x: (float) The factor by which the canvas is scaled.
+
+**Example(s):**
+
+*Scale horizontal and vertical canvas units by a factor of 2*
+
+```python hl_lines="6"
+%%ignite
+def setup():
+    size(400, 400)
+    circle(100, 100, 100)
```

Comment:
This extra circle just complicates the example. Also this one is completely hidden by the scaled up example... (remember to edit the highlight line `hl_lines` accordingly)
```suggestion
```